stranger/utils.py

```diff
@@ -236,6 +249,10 @@ def get_trgt_repeat_res(variant_info, repeat_info):
             for allele in mc.split(","):
                 mcs = allele.split('_')
                 # GT would have the index of the MC in the ALT field list if we wanted to be specific...
+                
+                # What should we do if MC is . ? 
```

Comment:
Ah, I hadn't seen that one? Is that from TRGT or sth from STRdust? Do they mean completely missing from motifs, or is it a shorthand for as-reference? ExHu was kind enough to calculate and pass the reference size given the current locus definitions, but I guess for TRGT we would perhaps pass it from the repeat definition file.
We should check what the callers intended here I think!

Reply:
This is for TRGT. Probably when the repeat could not be called. Some examples:

```
chr16   17470908        .       GCCGCCGCCGCCGCCC        .       0       .       TRID=XYLT1;END=17470923;MOTIFS=GCC;STRUC=(GCC)n        GT:AL:ALLR:SD:MC:MS:AP:AM       .:.:.:.:.:.:.:.
chr16   24613440        .       TTTTATTTTATTTTATTTTATTTTATTTTATTTTATTTTATTTTATTTTATTTTATTTTATTTTATTTTATTTTATTTTATTTTATTTTAT    .       0       .       TRID=TNRC6A;END=24613530;MOTIFS=TTTTA,TTTCA;STRUC=(TTTTA)n(TTTCA)n(TTTTA)n      GT:AL:ALLR:SD:MC:MS:AP:AM      .:.:.:.:.:.:.:.
chr16   66490399        .       TAAAATAAAATAAAATAAAATAAAATAAAATAAAATAAAATAAAATAAAATAAAATAATAAAATAAAAA   .       0              .       TRID=BEAN1;END=66490467;MOTIFS=TGGAA,TAAAA;STRUC=(TGGAA)n(TAAAA)n       GT:AL:ALLR:SD:MC:MS:AP:AM              .:.:.:.:.:.:.:.
```

Reply:
Ahoy! Yes, could be missing or reality saying hi with some unexpected alleles. I guess we also have to treat it as no-call rather than reference. It's unfortunate that TRGT does seem to set `QUAL==0` and no `FILTER=='.'` for everything. For ExHu we did set the size to 0 for `.` so I guess we might as well here!

Reply:
I run a test-dataset with reads that only covers a few of the pathogenic repeats.